nb2/bot/slack_bot.py

```diff
@@ -308,7 +334,20 @@ def is_remind_action(self, command: str) -> bool:
           tag  <@user_id_to_remind>
         - Only one <@user_id_to_remember> is allowed.
         """
-        regex_pattern = "^remind\\W+(me|(<@.*>)+)\\W+of\\W+<@.*>\\W*$"
+        regex_pattern = "^remind\\s+(me|(<@.*>)+)\\s+of\\s+<@.*>\\s*$"
+        return re.match(regex_pattern, command, re.I) is not None
+
+    def is_random_action(self, command: str) -> bool:
+        """
+        Return True if the command string indicates a call to NB's "random" quote action.
+
+        A valid "random" action is a command of the form:
+        'random quote'
+
+        Notes:
+        - Only one target user is allowed.
+        """
+        regex_pattern = "^random\\s+quote$"
```

Comment:
What's the difference between `\W` and `\s`?

Reply:
I was confused last time when I thought `\W` was whitespace.

`\W` is actually "not a word character". So it will match on whitespace, but will also match on like `<`

`\s` is "whitespace characters". So it should match on spaces, tabs, etc